web/forms.py

```
@@ -1517,6 +1519,37 @@ class StudentEnrollmentForm(forms.Form):
+            "title": TailwindInput(attrs={"required": True}),
+            "description": TailwindTextarea(attrs={"required": True}),
+            "date": TailwindDateTimeInput(attrs={"type": "datetime-local", "required": True}),
+            "link": TailwindInput(attrs={"type": "url"}),
+            "location": TailwindInput(),
+            "event_type": TailwindSelect(
+                attrs={"required": True},
+                choices=[("online", "Online"), ("in_person", "In Person")],
+            ),
+        }
+
+    def clean(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `clean`

(ANN201)